### ISSUE TITLE
[Feature request] Keep notes in sync

### ISSUE THREAD

**Issue by thelazyoxymoron** [benjypng/logseq-zoterolocal-plugin#24]
Thank you for such an amazing plugin!

I wanted to see if there's a way for the plugin to keep the notes in sync. My workflow is:

1. Add an article in Zotero using browser extension. This creates the snapshot of the webpage as well.
2. Open the article in Zotero and read/annotate some parts of it.
3. Pull in the article in Logseq and schedule it to be read later using [this incremental reading plugin](https://github.com/mochar/logseq-incremental-blocks).
4. Re-visit the article in Zotero when scheduled. Read some more and annotate.
5. The notes section for that article in Logseq should fetch the new note.

**Comment by benjypng** [comment]
Thanks! Actually 'syncing' or handling changes in Zotero within Logseq is something that has been built, but I've been waiting to actually release it because I don't have enough information to know if the way I've chosen to handle it actually solves the problem.

I think in your use-case, it is probably simple enough. But I've heard of others being more complex such as wanting to handle duplicates when annotations are done in both Logseq and Zotero, or whether only notes get synced but everything else doesn't, etc.

**Comment by thelazyoxymoron** [comment]
Makes sense. Perhaps an early version can be released with the mentioned caveats and disabled by default?
